src/content/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-apm-agents.mdx

```diff
@@ -21,158 +21,1190 @@ redirects:
   - /docs/logs/log-monitoring/enable-logs/configure-logs-context-apm-agents
 ---
 
-Logs in context makes it easy to link to your log data with related data across the rest of the New Relic platform. For example, you can easily correlate log messages to a related error trace or distributed trace in [APM](/docs/apm/applications-menu/monitoring/apm-overview-page-view-transaction-apdex-usage-data). This is accomplished by appending trace IDs to the corresponding application logs, automatically filtering to those logs from the [errors](/docs/apm/applications-menu/error-analytics/error-analytics-explore-events-behind-errors) or [distributed trace](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data) UIs.
+When you need to correlate log data with other telemetry data, enable logs in context in New Relic. Logs in context adds metadata that links your log data with related data, like [error](/docs/apm/apm-ui-pages/error-analytics/errors-page-find-fix-verify-problems/) or [distributed trace](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data) information in New Relic APM.
```

Comment:
"Logs in context adds metadata that links your log data with related data" - a lot of data in this sentence, is there a way to use different wording?